app/resources/linked_account_resource.rb

```diff
@@ -0,0 +1,17 @@
+class LinkedAccountResource < BaseResource
+  model_hint model: LinkedAccount::MyAnimeList
```

Comment:
List import has it not as a string. Docs are the same. So the way I have it is right.